angular-dfp.js

```diff
@@ -156,6 +168,13 @@ angular.module('ngDfp', [])
       enabled = setting;
     };
 
+    /**
+     Stores page targeting key->values
+     */
+    this.setPageTargeting = function () {
```

Comment:
I think this would be better if it was explicit (ie. `function (key, value)`). What do you think?